src/loadArticles.js

```diff
@@ -0,0 +1,28 @@
+const main = document.querySelector('main');
+
+const loadArticles = (data) => {
+  data.forEach((maze) => {
+    const article = document.createElement('article');
+    const img = document.createElement('img');
```

Comment:
As i said previously now we can use the previous function 

getData() to use it inside your loadArticles function 

`const loadArticles = () => {
  const data = await getData( )
  data.forEach((maze) => {
    const article = document.createElement('article');
    const img = document.createElement('img');`